examples/step-21/step-21.cc

```diff
@@ -241,29 +223,19 @@ namespace Step21
     {}
 
     virtual double value(const Point<dim> & p,
-                         const unsigned int component = 0) const override;
+                         const unsigned int component = 0) const override
+    {
+      return Functions::ZeroFunction<dim>(dim + 2).value(p, component);
```

Comment:
Well, that's about the most complicated way to write a zero :-)

Reply:
yup